webapp/app/actions.js

```diff
@@ -39,8 +27,11 @@ export const initApp = () => async dispatch => {
       survey,
     })
   } catch (error) {
-    dispatch({ type: appPropsChange, status: AppState.appStatus.ready, i18n })
-    CognitoAuth.logout()
+    dispatch({
```

Comment:
how come if user is not present, there's an exception ?

Reply:
right, the error is never thrown, I'll fix this.